packages/rocketchat-channel-settings-mail-messages/server/methods/mailMessages.coffee

```diff
@@ -0,0 +1,44 @@
+Meteor.methods
+	'mailMessages': (data) ->
+		if not Meteor.userId()
+			throw new Meteor.Error('invalid-user', "[methods] mailMessages -> Invalid user")
+
+		check(data, Match.ObjectIncluding({ rid: String, to: String, subject: String, messages: [ String ], language: String }))
+
+		room = Meteor.call 'canAccessRoom', data.rid, Meteor.userId()
+		unless room
+			throw new Meteor.Error('invalid-room', "[methods] mailMessages -> Invalid room")
+
+		rfcMailPatternWithName = /^(?:.*<)?([a-zA-Z0-9.!#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*)(?:>?)$/
+		emails = data.to.trim().split(',')
+		for email in emails
+			unless rfcMailPatternWithName.test email.trim()
+				throw new Meteor.Error('invalid-email', "[methods] mailMessages -> Invalid e-mail")
+
+		user = Meteor.user()
+		name = user.name
+		email = user.emails?[0]?.address
+
+		if data.language isnt 'en'
+			localeFn = Meteor.call 'loadLocale', data.language
+			if localeFn
+				Function(localeFn)()
+				moment.locale(data.language)
```

Comment:
Here are you changing the locale forever?
What happens if you send a mail in PT then another one in EN?